redhat-appstudio/remote-secret
# Support removal in label/annotations on remote secrets

### What does this PR do?
Support tracking of applied labels and annotations so that we can reliably track their removals in the target secrets.

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/SVPI-678

### How to test this PR?
1. Define a remote secret with some labels and annotations in the secret spec.
2. Upload the data to the remote secret and define a target so that the target secret is created.
3. Check that the labels and annos are present on the target secret.
4. Remove some of the labels/annos from the remote secret spec.
5. Check that the removed labels/annos are no longer present on the target secret.
